spec/models/consent_form_spec.rb

```diff
@@ -634,6 +634,35 @@
     expect(consent_form.health_answers).to be_empty
   end
 
+  it "combines health questions from multiple active vaccines" do
+    programme1 = create(:programme, :menacwy)
+
+    # Menacwy vaccine is discontinued in yml, we need >1 active vaccines to test
+    programme1.vaccines.first.update!(discontinued: false)
```

Comment:
I think this points to an issue in the config, the `MenQuadfi` should be active as it's the one the SAIS organisation will be vaccinating with.

Reply:
Yeah I've just checked, it's wrong.